nautobot/__init__.py

```diff
@@ -11,6 +11,21 @@
 # Sentinel to make sure we only initialize once.
 __initialized = False
 
+
+def add_success_logger():
+    """Add a custom log level for success messages."""
+    SUCCESS = 25
```

Comment:
Should 25 be an environment variable or a settings variable?

Reply:
This happens before settings are loaded, it would have to be an environment variable, but care would need to be taken to ensure that it does not use the same level as the Logging standard levels.

Reply:
Usually you don't change the levels, you are not going to change any of the Python standard levels. I believe 25 is a good level number, because if you have INFO, you probably want SUCCESS, but if you only want WARNING and ERRORS, your not going to see the Successes in your logging handler (like if you forward your logs to splunk). This also allows the LoggingHandlers that are already configured for INFO to also pick up the new SUCCESS logs.

Reply:
Agreed - this doesn't need to be user definable.